.pre-commit-config.yaml

```diff
@@ -20,6 +20,10 @@ repos:
     rev: 5.9.3
     hooks:
       - id: isort
+  - repo: https://github.com/pycqa/pylint
+    rev: v2.12.2
+    hooks:
+      - id: pylint
```

Comment:
I think we'll need the `local:` approach documented in the docs (I linked it above) due to the extra deps we have.

Reply:
Thanks @nejch  That was the key and a few other things.